src/components/Navbar/styles.scss

```diff
@@ -7,11 +7,27 @@
     justify-content: center;
   }
 }
-
+.navbar-toggle {
+  color: aliceblue;
+  padding: 2em;
+  top: 0;
+  left: 0px;
+  position: absolute;
+  display: fixed;
+}
 .navbar__list {
-  margin: 0 10px 0 10px;
-  padding: 20px;
+  flex-direction: column;
   color: white;
+  position: absolute;
+  top: 5%;
+  left: 0;
+  padding: 1rem 0 0 2rem;
+  display: flex;
+  width: 30vw;
```

Comment:
The issue with this measurement depending on the width of the screen is that mobile will look very tight. Maybe you could try two different width measures depending on a media query.

![image](https://user-images.githubusercontent.com/65427911/135704877-1c21d036-1b78-4fb9-82e5-4d5ec7d87967.png)

Reply:
ok let me try again